src/Symfony/Bundle/FrameworkBundle/Tests/Fixtures/Descriptor/event_dispatcher_1_event1.txt

```diff
@@ -1,8 +1,8 @@
 <info>[event_dispatcher]</info> Registered listeners for event <info>event1</info>
 
-+-------+-------------------+
-| Order | Callable          |
-+-------+-------------------+
-| #1    | global_function() |
-| #2    | \Closure()        |
-+-------+-------------------+
++-------+-------------------+----------+
+| Order | Callable          | Priority |
++-------+-------------------+----------+
```

Comment:
We could just replace the "Order" column with priority don't you think ? The "Order" column was just there because it was impossible to get priorities in dispatcher !